benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadWorker.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2016, Google Inc. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ *    * Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *    * Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following disclaimer
+ * in the documentation and/or other materials provided with the
+ * distribution.
+ *
+ *    * Neither the name of Google Inc. nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.grpc.benchmarks.driver;
+
+import io.grpc.Status;
+import io.grpc.benchmarks.proto.Control;
+import io.grpc.benchmarks.proto.WorkerServiceGrpc;
+import io.grpc.internal.ServerImpl;
+import io.grpc.netty.NettyServerBuilder;
+import io.grpc.stub.StreamObserver;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A load worker process which a driver can use to create clients and servers. The worker
+ * implements the contract defined in 'control.proto'.
+ */
+public class LoadWorker {
+
+  private static final Logger LOG = Logger.getLogger(LoadWorker.class.getName());
+
+  private final int serverPort;
+  private final ServerImpl driverServer;
+
+  LoadWorker(int driverPort, int serverPort) throws Exception {
+    this.serverPort = serverPort;
+    this.driverServer = NettyServerBuilder.forPort(driverPort)
+        .directExecutor()
+        .addService(WorkerServiceGrpc.bindService(new WorkerServiceImpl()))
+        .build();
```

Comment:
since the `driverServer` doesn't need to be able to handle much load, it might make sense to limit it to 1 thread, in order to reduce the number of threads i.e. create a eventloop that's shared by boss and workers.